### ISSUE TITLE
Support more models

### ISSUE THREAD

**Issue by siddharth9820** [axonn-ai/yalis#14]
Gemma models require non-standard variants of attention like softmax logit capping. This is currently not supported in yalis.
